README.md

```diff
@@ -6,27 +6,392 @@
+environment variables (see `.env.example`).
+
+If you want to run background tasks synchronously without Celery, set the environment variable
+`CELERY_ENABLED` to `False`. Scheduled tasks still need the worker in order to run.
+
+### Authentication
+
+Authentication is handled by [Helsinki Tunnistus] Keycloak using the [django-helusers] library.
+You'll need to get the `TUNNISTAMO_ADMIN_SECRET` from the [Azure Pipelines library]
+or from a colleague and set that in your `.env` file.
+
+[Helsinki Tunnistus]: https://helsinkisolutionoffice.atlassian.net/wiki/spaces/KAN/pages/419463195/
+[django-helusers]: https://github.com/City-of-Helsinki/django-helusers
+[Azure Pipelines library]: https://dev.azure.com/City-of-Helsinki/tilavarauspalvelu/_library?itemType=VariableGroups
+
+Authentication in application is managed using session cookies.
+
+### Static files
+
+Static files are served by the [Whitenoise] package.
+These are all files that are not uploaded by the users in Django Admin pages.
+
+[Whitenoise]: https://whitenoise.evans.io/en/stable/
+
+Media files are served by the [uWSGI static files implementation], offloaded to threads.
+These are all files uploaded by users in Django Admin pages.
+
+[uWSGI static files implementation]: https://uwsgi-docs.readthedocs.io/en/latest/StaticFiles.html
+
+### Image cache
+
+In production, [Varnish cache] is used for reservation unit and purpose images.
+When new image is uploaded, existing images are removed from the cache.
+
+[Varnish cache]: https://varnish-cache.org/
+
+In settings there are four configurations:
+- `IMAGE_CACHE_ENABLED` = Toggle caching on/off
+- `IMAGE_CACHE_VARNISH_HOST` = Varnish hostname
+- `IMAGE_CACHE_PURGE_KEY` = Secret key for doing purge requests
+- `IMAGE_CACHE_HOST_HEADER` = `Host` header value in purge request
+
+### Translations
+
+Translations are handled by Django's built-in translation system.
+GitHub Actions CI will check that all translations are up-to-date during PRs.
+To update translations, run `make translations`. This will update the `.po` files
+located in the `locale` directory.
+
+For model field translations, we use [django-modeltranslation].
+The package has integrations in all the relevant parts of the project
+(serializers, admin, etc.). See code for more details.
+
+[django-modeltranslation]: https://django-modeltranslation.readthedocs.io/en/latest/
+
+### Debugging
+
+For debugging during development, the [Django Debug Toolbar] package can be used.
+The [Django GraphQL Debug Toolbar] extension is used for the GraphQL endpoint.
+
+[Django Debug Toolbar]: https://django-debug-toolbar.readthedocs.io/en/latest/
+[Django GraphQL Debug Toolbar]: https://github.com/flavors/django-graphiql-debug-toolbar
+
+You should add a `local_settings.py` on the root level of the project and add
+three classes called `LocalMixin`, `DockerMixin` and `AutomatedTestMixin` to it.
+These can be used to override settings for local development and automated tests respectively.
```

Comment:
Refer to earlier section, which has the instructions to Copy `local_settings_example.py` to `local_settings.py`